tests/components/tplink/__init__.py

```diff
@@ -95,6 +101,64 @@
 }
 
 
+def _load_feature_fixtures():
+    fixtures = load_json_value_fixture("features.json", DOMAIN)
+    for fixture in fixtures.values():
+        if isinstance(fixture["value"], str):
+            try:
+                time = datetime.strptime(fixture["value"], "%Y-%m-%d %H:%M:%S.%f%z")
+            except ValueError:
+                pass
+            else:
+                fixture["value"] = time
```

Comment:
If I'm reading this correctly, this iterates over all the fixtures and tries to convert each of them to datetime, so this could probably be already done inside the try-block?

Reply:
Moved